public/survey-create.html

```
@@ -0,0 +1,453 @@
+
+  <main class="flex-1 max-w-4xl mx-auto w-full px-4 md:px-6 py-10 space-y-6">
+
+    <div id="login-warning" class="hidden rounded-2xl p-6 bg-amber-50 dark:bg-amber-900/20 border border-amber-200 dark:border-amber-800 text-amber-800 dark:text-amber-300 flex items-center gap-3">
+      <i class="fas fa-circle-exclamation text-xl"></i>
+      <span>You need to <a href="/login.html" class="underline font-semibold">log in</a> to create a survey. Your draft will be kept.</span>
+    </div>
+
+    <section class="bg-white dark:bg-gray-800 rounded-2xl shadow-md border border-gray-200 dark:border-gray-700 p-6 space-y-5">
+      <div>
+        <label class="block text-sm font-semibold text-gray-700 dark:text-gray-300 mb-1.5">Survey Title</label>
+        <input id="survey-title" type="text" maxlength="200" placeholder="e.g. Customer Satisfaction Survey" class="field-input" />
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)